### PR TITLE
Mobile Skeleton Project exercise 

### PR DESCRIPTION
**Pull request by Alfred-KInyua**
In this project, I have set up a "Mobile portfolio" repository.
The required tasks include.

Creating a new branch called header-feature according to GitHub flow requirements.
Setup linter to flag any errors or bugs.
Added style.css page, styling my mobile repository HTML page
Added the .gitignore file that contains the node_modules, which should be ignored by git.
Added index.html file and used Flexbox.
Added the MIT file to grant permissions and indemnify developers for future use.
Added a professional README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <title>My portfolio</title>
+</head>
+<body>
+    
+    <header>
+        <ul class="top mobile">
+            <li> <a href="#">9:15</a> </li>
+            <li> <a href="#"><img src="images/wifi.png" alt=""></a></li>        
+        </ul>
```
Comment (aimemalaika) [inline index.html:17]: - [ ] [OPTIONAL] Kindly remove this section as it is not part of the design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <ul class="top mobile">
+            <li class="welcome"> <a href="#"> welcome</a></li>
+            <li> <a href="#"> <img src="images/Union.svg" alt=""></a></li>
```
Comment (aimemalaika) [inline index.html:20]: - [ ] Put the hamburger button is on the right and has a nav tag as requested in the project requirements

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Alfred-KInyua, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfred-KInyua** [comment]
@aimemalaika 
Thanks for the review.

**Comment by Alfred-KInyua** [comment]
@aimemalaika 
I have made the following changes.

1. Removed the top part as requested.
2. Placed the hamburger button to the right of my mobile page.

File: index.html (hunk 1 above)
Comment (arslanbisharat) [reply index.html:17]: - [ ] [Optional] This change has not been implemented.

File: index.html (hunk 2 above)
Comment (arslanbisharat) [reply index.html:20]: - [ ] This change has not been implemented as requested by the previous CR.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Alfred-KInyua,
__
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfred-KInyua** [comment]
@arslanbisharat 
OHH silly me!!! I Forgot to push. Sorry

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Alfred-KInyua, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**